TestCases/py_wrapper/translating_NACA0012/config.cfg

```diff
@@ -43,18 +43,18 @@ MARKER_DEFORM_MESH= ( airfoil )
 % DISCRETIZATION METHODS
 %
 CONV_NUM_METHOD_FLOW= ROE
-MUSCL_FLOW= NO
+MUSCL_FLOW= YES
 NUM_METHOD_GRAD= GREEN_GAUSS
 SLOPE_LIMITER_FLOW= VENKATAKRISHNAN_WANG
 VENKAT_LIMITER_COEFF= 0.1
 
 % SOLUTION ACCELERATION
 %
-CFL_NUMBER= 1000
+CFL_NUMBER= 10
```

Comment:
I can't make this case work unless I reduce the CFL substantially...
I tried putting back the residual contribution but it does not help.